server/handlers_v2.go

```diff
@@ -51,6 +51,13 @@ const (
 	// OkMsg is in status field with HTTP 200 response
 	OkMsg       = "ok"
 	selectorStr = "selector"
+	// StatusProcessed is a message returned for already processed reports stored in Redis
+	StatusProcessed = "processed"
+	// RequestsForClusterNotFound is a message returned when no request IDs were found for a given clusterID
+	RequestsForClusterNotFound = "Requests for cluster not found"
```

Comment:
"No requests found for given cluster" sounds more accurate than ""Requests for cluster not found"". WDYT?

Reply:
+1

Reply:
@tisnik is someone already using the mock API to expect this? I was following these messages from the mock so that it's not different in case someone already expects it.

Reply:
@Bee-lee not now, as IO team work on on-demand data gathering is a bit delayed. We can use the proposed message, not a problem at all.

Reply:
ok thanks! done